app/sass/techradar.scss

```diff
@@ -1,29 +1,24 @@
-@import url(https://fonts.googleapis.com/css?family=Lato:100,300,400);
-@import url(https://fonts.googleapis.com/css?family=Handlee);
-
 $color_orange: rgb(224, 129, 0);
 $color_light_orange: #ff8917;
 
 $break_xxl: 1477px;
 $break_xl: 1280px;
 
+body{
+  margin:0;
+  background: #ffffff;
+}
 
 #simplemodal-overlay { background-color: rgb(51, 51, 51); }
 #simplemodal-container {background-color:#333; border:8px solid #444; padding:12px;}
 
 #main {
-  font-family: 'Lato', sans-serif;
+  font-family: 'Arial', sans-serif;
```

Comment:
The Lato had issues displaying on Windows Chrome and Firefox:
https://www.dropbox.com/s/hoa6hglnw9bt6ff/Screenshot%202016-06-06%2010.37.18.png?dl=0